src/pyhf/infer/test_statistics.py

```diff
@@ -226,3 +226,56 @@ def tmu_tilde(mu, data, pdf, init_pars, par_bounds):
             + 'Use the tmu test statistic (pyhf.infer.test_statistics.tmu) instead.'
         )
     return _tmu_like(mu, data, pdf, init_pars, par_bounds)
+
+
+def q0(data, pdf, init_pars, par_bounds):
```

Comment:
`q0` still needs a separate function since it needs to clip for `muhat < 0` instead of `muhat > mu` as done in `_qmu_like`